_sass/_team.scss

```diff
@@ -37,22 +37,43 @@
 .team-member-content{
     @media (max-width: $desktop-width) {
         > h3 {
-            font-size: 1.5em;
+            font-size: 1em;
+            word-wrap: break-word;
         }
 
         > .team-member-position {
-            font-size: 1.25em;
+            font-size: .75em;
+            word-wrap: break-word;
         }
     }
     @media (min-width: $desktop-width) {
         > p {
             margin: 0;
+            word-wrap: break-word;
         }
     }
 }
 
 .team-profile {
     @media (max-width: $desktop-width) {
         margin: 50px auto;
+        padding-top: 50vh;
```

Comment:
Is there a reason you've added this padding? It's a lot of space, and makes the profile look a little weird on mobile